cdmtaskservice/condor/client.py

```diff
@@ -144,3 +210,65 @@ async def run_job(self, job: models.Job) -> int:
         # could probably make itemdata an generator, YAGNI
         jobres = await asyncio.to_thread(self._schedd.submit, sub, itemdata=iter(itemdata))
         return jobres.cluster()
+    
+    # TODO CANCEL_JOB for condor
+
+    async def get_container_status(self, cluster_id: int, container_number: int) -> dict[str, Any]:
+        """
+        Get the htcondor status for a specific container for a job.
```

Comment:
What does "container_number" mean in this context?  Is this how we are naming the jobs?

Reply:
container number makes me think of the docker container ID

Reply:
So what is a container?

Reply:
Is it one job = one container or one job has multiple containers in it?

Reply:
A container is a Docker container. One job can have many containers

> if there is multiple containers in one job, you would presumably look up the job rather than the container

That's what get_job_status does

> This is a vanilla universe job right?

Yeah, that's the default if you don't specify the universe is my understanding

Reply:
done